src/app/controllers/githubs_controller.rb

```diff
@@ -4,7 +4,7 @@ class GithubsController < ApplicationController
   def new
     user = User.find_or_create_from_github_user(github_user)
     session[:user] = user.id
-    redirect_to root_path
```

Comment:
is this really working? because I think back in this case is the github oauth page itself not our website page

Reply:
Nope. It works perfectly fine.